src/search/suggestions/suggestions.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import { cleanup, render } from '@testing-library/react';
+import { afterEach, describe, expect, it, vi } from 'vitest';
+
+import { createAutosuggestOptionsMock } from '../../mocks/autosuggest-options.mock';
+import { createSetupConfigMock } from '../../mocks/configuration.mock';
+import { Suggestions } from './suggestions';
+import { SuggestionsProps } from './suggestions.types';
+
+describe('Suggestions', () => {
+  afterEach(() => {
+    cleanup();
+    vi.clearAllMocks();
+  });
+
+  const configuration = createSetupConfigMock();
+  const suggestOptions = createAutosuggestOptionsMock();
+
+  const props: SuggestionsProps = {
+    configuration,
+    suggestOptions,
+    inputValue: 'chair',
+  };
+
+  it('should render successfully', () => {
+    const {
+      container: { firstElementChild },
+    } = render(<Suggestions {...props} />);
+
+    expect(firstElementChild).toBeTruthy();
```

Comment:
you can change this from `tobeTruthy` to `toMatchSnapshot` to catch any mis-rendering in the future

Reply:
yea makes sense